packages/api/src/index.ts

```diff
@@ -162,18 +166,28 @@ const getApp = async () => {
 				res.status(500).send(msg);
 				return;
 			}
-			const parsedItem = TranscriptionItem.safeParse(item);
+			const parsedItem = TranscriptionDynamoItem.safeParse(item);
 			if (!parsedItem.success) {
 				const msg = `Failed to parse item ${exportRequest.data.id} from dynamodb. Error: ${parsedItem.error.message}`;
 				console.error(msg);
 				res.status(500).send(msg);
 				return;
 			}
+			const transcriptText = await getObjectText(
+				s3Client,
+				config.app.transcriptionOutputBucket,
+				parsedItem.data.transcriptKeys.text,
+			);
+			if (!transcriptText) {
+				const msg = `Failed to export transcript - it is possible your transcript has expired. Please re-upload the file and try again.`;
+				res.status(500).send(msg);
```

Comment:
Good shout, I'm just trying to write this so we return a 410 if the file isn't there, but a 500 for any other S3 error, but otherwise will just return a 410 for everything

Reply:
thanks!

Reply:
I've updated this (still needs testing on CODE) - we now:
 - 404 if user doesn't own the file
 - 410 if the file doesn't exist on S3
 - 500 for any other errror downloading from S3